common/navbar.js

```diff
@@ -140,6 +192,14 @@
                 scope.toggleMenu = toggleMenu;
                 scope.onToggle = onToggle;
 
+                scope.canShow = function (item) {
+                    return canShow(item, dcctx.session);
```

Comment:
I might be wrong, but the only place that I saw that we're attaching `session` to the context object is [here in login.js](https://github.com/informatics-isi-edu/chaise/blob/master/common/login.js#L21), which is inside a callback. So while we're running this `canShow` and `canEnable` functions, the `dcctx.session` might not have the proper value. Is that correct? Shouldn't these functions take care of that?

This is not directly related to your changes, but why do we even need to attach the session to dcctx? Shouldn't any place that needs to look at the session value, call the `Session.subscribeOnChange`?  Shouldn't these two `canShow` and `canEnable` call the `Session.subscribeOnChange` as well??
Also even if we want to have `session` under that object, we should directly attach extra attributes to the result of `getContextJSON`, and instead, we should call an appropriate setter function.